<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FVMainWindow</class>
 <widget class="QMainWindow" name="FVMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Azure</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Alogo.jpg</normaloff>Alogo.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>500</width>
      <height>277</height>
     </rect>
    </property>
    <property name="title">
     <string>File Group</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Address">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>411</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>76</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Front</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FrontFileName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="BackFileName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>76</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Meshbtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibration</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StatisticButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Sumbtn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>110</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Summory</string>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>481</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>781</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Force Curve</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>781</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Histogram</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>X Down</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="ThresholdBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="Scaleup">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>175</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>600</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="DownBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>84</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>24</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>X Down</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="UpBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="YScale">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>205</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="ScaleDown">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>145</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>600</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>115</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>321</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <zorder>groupBox_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_9</zorder>
    <zorder>ThresholdBox</zorder>
    <zorder>Scaleup</zorder>
    <zorder>DownBox</zorder>
    <zorder>label_6</zorder>
    <zorder>label_4</zorder>
    <zorder>label_10</zorder>
    <zorder>label_8</zorder>
    <zorder>UpBox</zorder>
    <zorder>YScale</zorder>
    <zorder>ScaleDown</zorder>
    <zorder>label_7</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件&amp;F</string>
    </property>
    <addaction name="actionSelect_S"/>
    <addaction name="actionNew_N"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSelect_S">
   <property name="text">
    <string>Select(S)</string>
   </property>
  </action>
  <action name="actionNew_N">
   <property name="text">
    <string>New(N)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
